projects/adin1300_dual/common/adin1300_dual_bd.tcl

```diff
@@ -0,0 +1,53 @@
+create_bd_port -dir I ref_clk_125
+create_bd_port -dir O reset
+
+create_bd_port -dir O -from 1 -to 0 -type data speed_mode_a
+create_bd_port -dir O -from 1 -to 0 -type data speed_mode_b
+
+ad_ip_instance gmii_to_rgmii gmii_to_rgmii_0
+ad_ip_parameter gmii_to_rgmii_0 CONFIG.SupportLevel Include_Shared_Logic_in_Core
+
+# 200MHz for 7 series; 375 for ultrascale
+ad_ip_instance clk_wiz clk_wiz
```

Comment:
In both ZC706 and ZCU102 projects I see the following warning:
[Opt 31-32] Removing redundant IBUF since it is not being driven by a top-level port. i_system_wrapper/system_i/clk_wiz/inst/clkin1_ibuf 
Resolution: The tool has removed redundant IBUF. To resolve this warning, check for redundant IBUF in the input design.
Can you change the parameters of the clk_wiz to remove the input buffer ?